docs/configuration.rst

```diff
@@ -140,6 +140,9 @@ Mongodb
 - ``MONGODB_PORT`` (default: ``27017``)
 - ``MONGODB_USERNAME`` (default: ``""``)
 - ``MONGODB_PASSWORD`` (default: ``""``)
+- ``MONGODB_REPLICA_SET`` (default: ``""``)
+- ``MONGODB_USE_SSL`` (default: ``false``)
+- ``MONGODB_AUTH_SOURCE`` (default: ``""``)
```

Comment:
`MONGODB_PROTOCOL` is a new configuration variable which should be added here.

Reply:
Updated as requested